src/frontend/tsconfig.json

```diff
@@ -7,7 +7,7 @@
         "resolveJsonModule": true,
         "allowJs": true,
         "allowSyntheticDefaultImports": true,
-        "typeRoots": ["../../node_modules/@types", "./@types", "./global.d.ts"]
+        "typeRoots": ["../../node_modules/@types", "./@types"]
```

Comment:
Må legge på denne igjen, men har måttet gjøre dette for å få lov til å committe 🫠